packages/cli/README.md

```diff
@@ -36,47 +36,48 @@ Use `npm` to create a `package.json` file:
 npm init
 ```
 
-And initialize the ZeppelinOS project:
+And initialize the OpenZeppelin SDK project:
 
 ```sh
-zos init my-project
+openzeppelin init my-project
 ```
 
-Now it is possible to add contracts to the project with the `zos add` command,
-push these contracts to a blockchain network with `zos push`, use
-`zos create` to create instances for these contracts that later can be
+Now it is possible to add contracts to the project with the `openzeppelin add` command,
+push these contracts to a blockchain network with `openzeppelin push`, use
+`openzeppelin create` to create instances for these contracts that later can be
 upgraded, and many more things.
 
-Run `zos --help` for more details about this and all the other functions of
-ZeppelinOS.
+Run `openzeppelin --help` for more details about this and all the other functions of
+the OpenZeppelin CLI.
 
 The
-[ZeppelinOS documentation](https://docs.zeppelinos.org/)
-explains how to use the `zos` command-line interface to build a project, to
+[OpenZeppelin SDK documentation](https://docs.openzeppelin.com/sdk/2.4)
+explains how to use the `openzeppelin` command-line interface to build a project, to
 upgrade contracts and to share packages for other projects to reuse. It also
-explains how to operate the project with the ZeppelinOS JavaScript libraries
-instead of this `zos` command.
+explains how to operate the project with the OpenZeppelin JavaScript libraries
+instead of this `openzeppelin` command.
 
 ## Security
 
-If you find a security issue, please contact us at security@zeppelinos.org. We
+If you find a security issue, please contact us at security@openzeppelin.com. We
 give rewards for reported issues, according to impact and severity.
 
 ## Maintainers
 
-* [@facuspagnuolo](https://github.com/facuspagnuolo/)
 * [@spalladino](https://github.com/spalladino)
+* [@jcarpanelli](https://github.com/jcarpanelli)
+* [@ylv-io](https://github.com/ylv-io)
 
 ## Contribute
 
 To contribute, join our
 [community channel on Telegram](https://t.me/zeppelinos) where you can talk to
-all the ZeppelinOS developers, contributors, partners and users.
+all the OpenZeppelin developers, contributors, partners and users.
 
 You can also follow the recent developments of the project in our
-[blog](https://blog.zeppelin.solutions/) and
-[Twitter account](https://twitter.com/zeppelinorg).
+[blog](https://blog.openzeppelin.com/) and
+[Twitter account](https://twitter.com/openzeppelin).
```

Comment:
```suggestion
[Twitter account](https://twitter.com/ZeppelinOrg).
```

Reply:
I asked Orne which twitter should we use and she told me that we are moving it to `openzeppelin`. Should I commit this suggestion?

Reply:
Wow, didn't know that! Disregard my suggestion then, @ornellacordoba is the unique source of truth here.